### DISCUSSION TITLE
How to pass arguments to the webview?

### DISCUSSION THREAD

**Discussion by NikolaGrgic** [tauri-apps/tauri#2903] [Q&A]
Hi all,

I'd like to pass arguments to allow self-signed SSL certificates on the frontend. This will interact with a server I'll have on the backend. Is there any way to pass an argument to the webview to tell it to ignore self-signed certificates, possible even on a specific URL?

**Comment by tristanpoland** [comment]
I would like to second this. I need a way to raise the JS heap size and I know the flag I need, I just need to pass it to webview somehow.

**Reply by FabianLars** [reply]
https://v2.tauri.app/reference/config/#additionalbrowserargs (windows only - the other platforms' webviews don't have something like this)
